packages/containers/src/ObjectViewer/ObjectViewer.container.js

```diff
@@ -0,0 +1,92 @@
+import React, { PropTypes } from 'react';
+import Immutable, { List, Map } from 'immutable';
+import get from 'lodash/get';
+
+import { ObjectViewer as Component } from 'react-talend-components';
+import { statePropTypes, stateWillMount } from '../state';
+
+export const DEFAULT_STATE = new Map({
+	edited: new List(),  // Array of JSONPath
+	opened: new List(),  // Array of JSONPath
+	modified: new Map(),  // Store the onChange
+});
+
+class ObjectViewer extends React.Component {
+	static displayName = 'CMFContainer(ObjectViewer)';
+	static propTypes = {
+		id: PropTypes.string,
+		data: get(Component, 'propTypes.data'),
+		displayMode: get(Component, 'propTypes.displayMode'),
+		onSubmit: get(Component, 'propTypes.onSubmit'),
```

Comment:
Why do you use lodash `get` on those 3 lines instead of just getting it ? `Component.propTypes.onSubmit`